server/datastore/index.yaml

```diff
@@ -28,6 +28,5 @@ indexes:
   - name: ProjectID
   - name: SpecID
   - name: VersionID
-  - name: Currency
```

Comment:
Just realized we might need this still. I haven't fully eliminated currency from ListSpecs because some changes are necessary to list only the most recent revisions of specs without it. Locally everything seems to work fine, but without the previous index (with currency) there might be issues.

Reply:
Right - thanks for noticing. I restored the original index and added the new one